install/sonic/LoM-install.sh

```diff
@@ -0,0 +1,443 @@
+
+    if [[ $1 == 0 ]]; then
+        state="disabled"
+    elif [[ $1 == 1 ]]; then
+        state="enabled"
+    else
+        fail "Internal error in usage. Expect arg as 1 or 0" ${ERR_DB}
+    fi
+
+    # Create FEATURE table entry
+    RET="$(redis-cli -n 4 hmset "FEATURE|device-health" "auto_restart" "enabled" \
+        "delayed" "True" "has_global_scope" "True" "has_per_asic_scope" "False" \
+        "high_mem_alert" "disabled" "set_owner" "kube" "state" "${state}" \
+        "support_syslog_rate_limit" "true")"
+
+    [[ "${RET}" == "OK" ]] || { fail "failed to create/update FEATURE table entry for state=${state}" ${ERR_DB}; }
+
+    fEnd DBUpdate
+}
+
+
+
+function installCode()
+{
+    fStart installCode
+    pushd $(dirname $0)/../host
+
+    for i in ${HOST_FILES}; do
+        sudo cp $i /$i
+    done
+    popd
+
+    fl="$(dirname $0)/../install/${IMAGE_FILE}"
+    docker load -i ${fl}
+    [[ $? != 0 ]] && { fail "Failed to load docker image ${fl}" ${ERR_INSTALL_CODE}; }
+    tag="$(cat $(dirname $0)/VERSION |  tr -d '\n')"
+    docker tag ${IMAGE_NAME}:latest ${IMAGE_NAME}:${tag}
+    [[ $? != 0 ]] && { fail "Failed to tag ${IMAGE_NAME}:latest to ${tag}" ${ERR_INSTALL_CODE}; }
+
+    fEnd installCode
+}
+
+
+function rollBackCode()
+{
+    # Rollback to last back up version
+    # Remove current install, if any
+    # Rename back up files as current and backup image as latest
+    #
+    [[ ${image_backup} == 0 ]] && { fail "Backup don't exist. Nothing to rollback;" ${ERR_ROLLBACK}; }
+
+    fStart rollBackCode
+
+    # Remove current install
+    forceClean 1
+
+    pushd /
+    for i in ${HOST_FILES}; do
+        sudo mv $i.${BACK_EXT} $i
+        [[ $? != 0 ]] && { fail "Failed to rollback  mv $i.${BACK_EXT} $i;" ${ERR_ROLLBACK}; }
+    done
+    popd
+
+    # coins backup tag with BACK_EXT 
+    backup_tag "${image_backup_tag}"
+    docker tag ${IMAGE_NAME}:${backupTag} ${IMAGE_NAME}:latest
+    [[ $? != 0 ]] && { fail "Failed to tag ${IMAGE_NAME}:${BACK_EXT} ${IMAGE_NAME}:latest" ${ERR_ROLLBACK}; }
+
+    docker tag ${IMAGE_NAME}:latest ${IMAGE_NAME}:${image_backup_tag}
+    [[ $? != 0 ]] && { fail "Failed to tag ${IMAGE_NAME}:${BACK_EXT} ${IMAGE_NAME}:latest" ${ERR_ROLLBACK}; }
+
+    docker rmi ${IMAGE_NAME}:${backupTag}
+    [[ $? != 0 ]] && { fail "Failed to untag ${IMAGE_NAME}:${backupTag}" ${ERR_ROLLBACK}; }
+
+    fEnd rollBackCode
+}
+
+
+function serviceRestart()
+{
+    fStart serviceRestart
+
+    sudo systemctl daemon-reload
+    sudo systemctl enable device-health.service 
+    sudo systemctl reset-failed device-health.service 
+
+    # Create/Update FEATURE table entry.
+    #
+    DBUpdate 1
+
+    # Restart installed/upgraded/rolledback service instance.
+    #
+    sudo systemctl restart device-health.service 
+    [[ $? != 0 ]] && { fail "Failed to restart device-health service" ${ERR_RESTART}; }
+
+    # Pause for a minute
+    #
+    echo "Take a 1 min pause to settle down"
+    sleep 1m
```

Comment:
Isn't 1min large? There is a check for processes start next , can't we have timer based check like at 
https://github.com/GouthamBanala/sonic-device-health/blob/gou_working/lom/integration_test/src/api.py API :: wait_for_lom_services_to_start

Reply:
From my experience from Integration tests, time is less than min

Reply:
Better be safe. 
We can't afford to have any false positive.
Think of FUSE pipeline failure.